contracts/Core/parameters/child/StakeManagerParams.sol

```diff
@@ -14,12 +14,13 @@ abstract contract StakeManagerParams is ACL, IStakeManagerParams {
     uint8 public withdrawReleasePeriod = 5;
     uint8 public extendLockPenalty = 1;
     uint8 public maxCommission = 20;
+
     // change the commission by 3% points
     uint8 public deltaCommission = 3;
-    uint16 public override baseDenominator = 10000;
     uint16 public gracePeriod = 8;
     uint16 public epochLength = 300;
     uint16 public epochLimitForUpdateCommission = 100;
+    uint16 public constant BASE_DENOMINATOR = 10000;
```

Comment:
constants should be in https://github.com/razor-network/contracts/blob/master/contracts/Core/storage/Constants.sol

Reply:
@hrishikeshio BASE_DENOMINATOR is needed in governance.sol and stakeparams.sol due to which constants were being inherited twice

Reply:
@hrishikeshio Shall I put it in constants.sol?

Reply:
This should be fine. Add a comment here in the code explaining why it needs to be here.

Reply:
why is it needed in stakeManagerparams.sol ?

Reply:
I have added the comment in governance.sol